### PR TITLE
scripts/find updates: Add more components

### PR DESCRIPTION
**Pull request by surajssd**
This commit checks updates for prometheus-operator, rook, velero,
aws-ebs-csi-driver, cluster-autoscaler.

### CODE REVIEW

File: scripts/find-updates.sh [hunk 1]
```diff
@@ -142,3 +142,59 @@ version=$(grep version "${tmpdir}/openebs/Chart.yaml" | cut -d":" -f2)
...
+cd "${workdir}"
+current_version=$(grep appVersion ./assets/components/cluster-autoscaler/Chart.yaml | cut -d":" -f2 | sed 's/ //g')
+
+tmpdir=$(mktemp -d)
+cd "${tmpdir}"
+
+# helm repo add autoscaler https://kubernetes.github.io/autoscaler >/dev/null 2>&1
+# helm repo update >/dev/null 2>&1
+# helm fetch --untar --untardir ./ autoscaler/cluster-autoscaler
+# version=$(grep appVersion "${tmpdir}/cluster-autoscaler/Chart.yaml" | cut -d":" -f2 | sed 's/ //g')
```
Comment (surajssd) [inline scripts/find-updates.sh:197]: This is waiting on https://github.com/kubernetes/autoscaler/issues/3398

**Comment by invidian** [comment]
@surajssd rather than adding `do-not-merge` label, maybe convert it to draft so then it cannot be merged?

**Comment by surajssd** [comment]
Removed this snippet, we can add whenever we see that it has been resolved. It is taking a very long time.

```bash
###########################
# Cluster autoscaler
cd "${workdir}"
current_version=$(grep appVersion ./assets/components/cluster-autoscaler/Chart.yaml | cut -d":" -f2 | sed 's/ //g')

tmpdir=$(mktemp -d)
cd "${tmpdir}"

helm repo add autoscaler https://kubernetes.github.io/autoscaler >/dev/null 2>&1
helm repo update >/dev/null 2>&1
helm fetch --untar --untardir ./ autoscaler/cluster-autoscaler
version=$(grep appVersion "${tmpdir}/cluster-autoscaler/Chart.yaml" | cut -d":" -f2 | sed 's/ //g')

printf "${format}" "cluster-autoscaler" "${current_version}" "${version}"
rm -rf "${tmpdir}"
```

**Comment by invidian** [comment]
I'm fine with merging commented out code too.

**Comment by surajssd** [comment]
> I'm fine with merging commented out code too.

Filed an issue for this https://github.com/kinvolk/lokomotive/issues/809.

**Pull request merged**